src/js/disability-benefits/routes.jsx

```diff
@@ -12,6 +12,7 @@ import AskVAPage from './containers/AskVAPage.jsx';
 import DocumentRequestPage from './containers/DocumentRequestPage.jsx';
 import AdditionalEvidencePage from './containers/AdditionalEvidencePage.jsx';
 import ClaimEstimationPage from './containers/ClaimEstimationPage.jsx';
+import AppealLayout from './components/AppealLayout';
```

Comment:
It looks like AppealLayout is containing a container, should it also be a container rather than a component (located in the component directory)?

Reply:
I think since it's a regular component it should go in components.